### PR TITLE
modify loginfo when visiting namesrv

### PR DESCRIPTION
**Pull request by keranbingaa**
## What is the purpose of the change
work for this [issue](https://github.com/apache/rocketmq/issues/1477), the loginfo is ambiguity when visiting namesrv, so we modify the loginfo to make it clear.

## Brief changelog
use the variable channel to replace addr in the log statement.

### CODE REVIEW

File: remoting/src/main/java/org/apache/rocketmq/remoting/netty/NettyRemotingClient.java [hunk 1]
```diff
@@ -374,15 +374,15 @@ public RemotingCommand invokeSync(String addr, final RemotingCommand request, lo
                 doAfterRpcHooks(RemotingHelper.parseChannelRemoteAddr(channel), request, response);
                 return response;
             } catch (RemotingSendRequestException e) {
-                log.warn("invokeSync: send request exception, so close the channel[{}]", addr);
+                log.warn("invokeSync: send request exception, so close the channel[{}]", channel);
```
Comment (wqliang) [inline remoting/src/main/java/org/apache/rocketmq/remoting/netty/NettyRemotingClient.java]: I think `RemotingUtil.socketAddress2String(channel.remoteAddress())` would be better

**Comment by duhenglucky** [comment]
@keranbingaa address seems more clear than the channel, IMHO, print all channel info isn't a good choice in here.

**Comment by qqeasonchen** [comment]
@duhenglucky I modify what print out， and now it just print the address, and when the variable address is null, it will also print the remote address.

**Comment by keranbingaa** [comment]
@duhenglucky  I modify what print out， and now it just print the address, and when the variable address is null, it will also print the remote address.

**Comment by ChaosYjh** [comment]
I think printing address is more clear too.And just printing channel.remoteAddress() will be ok.

**Pull request closed without merging**